helm-charts/0.25.4/README.md.gotmpl

```diff
@@ -0,0 +1,104 @@
+# VMware Secrets Manager (VSecM) Helm Chart
```

Comment:
This is the template that generates the `README.md`.

I caried over as much docs from the original README as it makes sense.